go.mod

```diff
@@ -10,8 +10,9 @@ require (
 	github.com/go-openapi/spec v0.19.7
 	github.com/imdario/mergo v0.3.8
 	github.com/kubevirt/cluster-network-addons-operator v0.3.1-0.20200512200059-2229227b9dac
+	github.com/kubevirt/vm-import-operator v0.0.2-0.20200506145247-c753eb7a2259
```

Comment:
could you release go module using semver instead of a pseudo-version?

Reply:
this is how go mod updated it. The version is released here https://github.com/kubevirt/vm-import-operator/releases/tag/v0.0.2